network/host/dht.go

```diff
@@ -328,7 +310,11 @@ func (dht *DHT) Bootstrap() error {
 	}
 
 	wg.Wait()
+	err := dht.iterateHt(cb)
+	return err
```

Comment:
You could actually just return `dht.iterateHt(cb)` w/o extra assignment.

Reply:
fixed